### ISSUE TITLE
Preferences window design idea

### ISSUE THREAD

**Issue by musiclover382** [Taiko2k/Avvie#25]
I don't like how we add previews atm. Of course the current ux does its job but I don't like the way it looks, the entry and the button are both on the left and there is a lot of free space on the right. So my first thought was to move the Add button to the right. It stops looking out of place, however it's not the best imo.
![Avvie mockup v1](https://user-images.githubusercontent.com/95233098/166166000-f3b4f234-981f-44a6-b33f-2b37824af9e8.png)
So I thought we could integrate the Add preview option in the Appearance category, and to me it looks very good.
![Avvie mockup part 1](https://user-images.githubusercontent.com/95233098/166166320-006be53a-ddcf-41ca-896b-450cfb3dbe2f.png)
Once the user presses the "+" button, a toast indicates that the preview was added and the button becomes green. Also, even before pressing it, the button is already green if the size selected is already in the main window. Pressing the green button either does nothing or shows a toast indicating that that size is already in the main window. The toast could also show the size that was added, e.g. "64x64 preview added!".
![Avvie mockup part 2](https://user-images.githubusercontent.com/95233098/166166334-ddfc955b-8a20-41d1-8cf7-733b627bbbef.png)

**Comment by Taiko2k** [comment]
Certainly, your proposal looks better. Is this working code?

**Comment by musiclover382** [comment]
No, it’s just a mockup. Unfortunately I don’t have the skills to implement this myself 🥲

**Comment by rafaelmardojai** [comment]
I could help with this.

IMO the current implementation is a bit confusing. I don't know if would be better to show a `+` button next to the previews in the main window. Or at last, be able to see the current added previews and be able to remove them in the settings window.

**Comment by Taiko2k** [comment]
Yes it's a bit haphazard of a design currently. And is something users get confused about. It's not obvious to look in preferences for this option.

Being able to add them from the main window would be a UX improvement.

I might add it to the preview context menu. So if a user right clicks a preview to remove one, they'll see where to add one back again. And have this section of the preferences moved to a new dialog.
